### ISSUE TITLE
Trip leaders unable to see trips in draft

### ISSUE THREAD

**Issue by nickcedwards** [ctcit/triphub#80]
Trip leaders can return a trip that they have created to "Draft" if it isn't ready for approval yet, but once that happens they can no longer see it, and can't return it to Pending. As an admin, I can see those trips under "Drafts", but trip leaders cannot see this section. They should at least be able to see their own trips in Draft

**Comment by alastairgbrown** [comment]
This code in `TripStates.tsx` needs changing as below.  Due to recent react module upgrades, I'm currently unable to compile anything.
```diff
    Draft: {
        id: 'Draft',
        status: 'This trip is in Draft status',
        button: 'Set to draft',
        icon: 'fa-pencil-ruler',
        prompt: 'Please describe why the trip is being set to draft (optional)',
        label: 'Reason for being in draft',
        nextStates: ['Pending'],
        roleToChange: Role.TripLeader,
-       roleToView: Role.Admin,
+       roleToView: Role.TripLeader,
    },
```

**Comment by nickcedwards** [comment]
Fixed in 7fbde60
